packages/cli/src/databases/repositories/folder.repository.ts

```diff
@@ -144,7 +149,9 @@ export class FolderRepository extends Repository<Folder> {
 			});
 		}
 
-		if (filter?.parentFolderId) {
+		if (filter?.parentFolderId === '0') {
```

Comment:
Is this an agreed convention that `0` = no parent folder?

Reply:
Agreed with the FE for the folders feature.The other options were null (string), root or none? But neither is great. Thoughts?

Reply:
Maybe "none" is more easy to understand.

Reply:
I guess we can agree that `0` is the id of the "root" folder